qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -0,0 +1,161 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Converters for migration from IBM Quantum BackendV1 to BackendV2."""
+
+from typing import Any, Dict
+
+from qiskit.transpiler.target import Target, InstructionProperties
+from qiskit.providers.backend import QubitProperties
+from qiskit.utils.units import apply_prefix
+from qiskit.circuit.library.standard_gates import IGate, SXGate, XGate, CXGate, RZGate
+from qiskit.circuit.parameter import Parameter
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.measure import Measure
+from qiskit.circuit.reset import Reset
+from qiskit.providers.models.pulsedefaults import PulseDefaults
+from qiskit.test.mock.utils.json_decoder import decode_pulse_defaults
+
+
+def convert_to_target(
+    configuration: Dict, properties: Dict = None, defaults: Dict = None
+) -> Target:
```

Comment:
Consider using https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors for the commit that includes this file so that the original author (Matthew) is also represented in the history.